babyagi.py

```diff
@@ -83,7 +78,7 @@ def openai_call(prompt: str, model: str = OPENAI_API_MODEL, temperature: float =
         return response.choices[0].text.strip()
     else:
         # Use chat completion API
-        messages=[{"role": "user", "content": prompt}]
+        messages=[{"role": "system", "content": prompt}]
```

Comment:
What's the immediate effect of this? I get it what the semantic difference is, but not sure how it affects the execution.

Reply:
I don't think it has any significant immediate effects since we only send that one message. I've tried it and saw no noticeable change in behavior.
If I understood correctly, the model could treat user and system messages differently, so directives declared by system messages could be followed over user-defined ones. But I'm not sure if it does that already. 
I just felt using the system role for this prompt is more correct and might be beneficial if we start having longer conversations or taking user input at some point.